### ISSUE TITLE
Unable to easily change the icon color using a class property.

### ISSUE THREAD

**Issue by mariansimecek** [ng-icons/ng-icons#146]
**Describe the bug**
Starting with version v29.6.0 and the introduction of [icon color variable](https://github.com/ng-icons/ng-icons/pull/145), it is no longer possible to change the color of an icon using a class with the color property alone. You now need to use the `!important` keyword or apply the class to a wrapping element.

**To Reproduce**
1. Declare class with color property
```css
.color-icon {
    color: red
}

```
2. Use class with color property on ng-icon
```html
<ng-icon class="color-icon" name="..."/>
```
3. Icon is not red

**Expected behavior**
Be able to use class with color property.

**Screenshots**
![image](https://github.com/user-attachments/assets/5b400170-9c84-463c-85f6-24cc3748aa40)

**Browser (please complete the following information):**
- Google chrome - 130.0.6723.91

**Packages (please complete the following information):**
```json
  "@ng-icons/core": "^29.6.0",
  "@ng-icons/font-awesome": "^29.6.0",
```

**Comment by ashley-hunter** [comment]
Thank you very much for the report, I will get this fixed for you later today.

**Comment by mariansimecek** [comment]
@ashley-hunter I'm not sure what is the proper solution. There's no need to hurry. I've downgraded the version for now. Thanks for the amazing library!

**Comment by ashley-hunter** [comment]
> Thanks for the amazing library!

No problem! Appreciate your support!

> I'm not sure what is the proper solution.

I believe I can move the color style into a CSS layer to always prioritise user styles if they are set. That should resolve this issue. I had tested when the color was set in component styles as they have a higher specificity, but I omitted to check when set within a global stylesheet.

**Comment by ashley-hunter** [comment]
This should now be fixed in 29.6.1, if you have any further issues just let me know! Thank you.

**Comment by ross1296** [comment]
> This should now be fixed in 29.6.1, if you have any further issues just let me know! Thank you.

Has this reverted at all? I'm finding difficulty using TailwindCSS's utility class to change the colour, and it appears the ng-icon layer takes priority:

![Image](https://github.com/user-attachments/assets/637de52d-8193-4f10-84f4-5caa8fc27a5e)

**Comment by ashley-hunter** [comment]
Thank you for this, this will be very helpful for others with a similar issue in future!
